src/database/db_build.sql

```diff
@@ -0,0 +1,26 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users,books;
+-- Table user
+CREATE TABLE IF NOT EXISTS users (
+    id          SERIAL        PRIMARY KEY,
+    name        TEXT          NOT NULL,
+    email       VARCHAR(200)  NOT NULL,
+    password    VARCHAR(32)   NOT NULL
+);
+
+INSERT INTO users (name,email,password) VALUES ('samar','samar@gmail.com','samar123');
+
+-- Table books
```

Comment:
I would remove these comments (-- Table user etc) as it's clear what the tables are from their names.

Reply:
i remove the comments 
and the insert it's just for testing

and we will put the website in heroku